c2rust-transpile/src/translator/assembly.rs

```diff
@@ -752,7 +752,7 @@ impl<'c> Translation<'c> {
                     // Extract expression
                     let in_expr = in_expr.map(|(i, operand)| (i, operand.expression));
 
-                    // For inouts, change the dirspec to include 'in'
```

Comment:
NAK. An inout is not an input, it's a bidirectional (in and out) argument; see above where we also mention that we are handling inout arguments.

Reply:
Oh whoops I didn't see that.